config/routes.rb

```diff
@@ -4,4 +4,6 @@
   get "/about", to: "static_pages#about"
   get "/contact", to: "static_pages#contact"
   get "/signup", to: "users#new"
+  post "/signup",  to: "users#create"
+  resources :users
```

Comment:
only register method which you use. example 
```
resources :users, only: :show
```
because method new and create have been registered